src/polyfill.native.ts

```diff
@@ -49,10 +49,12 @@ class MediaQuery {
 
   // @ts-ignore
   public get matches(): boolean {
+    const windowDimensions = Dimensions.get("window");
     return mediaQuery.match(this.query, {
       type: "screen",
       orientation: this.orientation.toLowerCase(),
-      ...Dimensions.get("window")
```

Comment:
We might still need to pass `scale` through.

Reply:
True, even though it may not be used 🙂 On it!

https://github.com/ericf/css-mediaquery/blob/34ba6343938e72b270205447b297611c40140a4f/index.js#L42-L74